### ISSUE TITLE
Support for Crystal

### ISSUE THREAD

**Issue by alexanderadam** [CoatiSoftware/SourcetrailDB#12]
[Crystal](https://crystal-lang.org/) is inspired by [Ruby](https://github.com/CoatiSoftware/SourcetrailDB/issues/11) so I guess they might share some logic in a custom indexer.

**Comment by mlangkabel** [comment]
Yes, and if the Crystal indexer would be written in Ruby, those two may share the same SourcetrailDB bindings.

**Comment by alexanderadam** [comment]
True, and if it would be written in Crystal they could as well and it wouldn't require any additional interpreter. :wink:
